### PR TITLE
docs: rename managed to cloud

### PR DESCRIPTION
**Pull request by dmitryk-dk**
### Describe Your Changes

These changes were made as part of the title transition from Managed VictoriaMetrics to VictoriaMetrics Cloud

### Checklist

The following checks are **mandatory**:

- [x] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

File: docs/enterprise.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The use of VictoriaMetrics Enterprise components is permitted in the following c
 - Production use if you have a valid enterprise contract or valid permit from VictoriaMetrics company.
   Please contact us via [this page](https://victoriametrics.com/products/enterprise/) if you are intereseted in such a contract.
 
-- [Managed VictoriaMetrics](https://docs.victoriametrics.com/managed-victoriametrics/) is built on top of VictoriaMetrics Enterprise.
+- [VictoriaMetrics Cloud](https://docs.victoriametrics.com/victoriametrics-cloud/) is built on top of VictoriaMetrics Enterprise.
```
Comment (AndrewChubatiuk) [inline docs/enterprise.md:31]:
```suggestion
- [VictoriaMetrics Cloud](./victoriametrics-cloud/README.md) is built on top of VictoriaMetrics Enterprise.
```

File: docs/guides/README.md [hunk 2]
```diff
@@ -6,6 +6,6 @@
 1. [Migrate from InfluxDB to VictoriaMetrics](migrate-from-influx.html)
 1. [Multi-regional setup with VictoriaMetrics: Dedicated regions for monitoring](multi-regional-setup-dedicated-regions.html)
 1. [How to delete or replace metrics in VictoriaMetrics](guide-delete-or-replace-metrics.html)
-1. [How to monitor kubernetes cluster using Managed VictoriaMetrics](/managed-victoriametrics/how-to-monitor-k8s.html)
+1. [How to monitor kubernetes cluster using VictoriaMetrics Cloud](/victoriametrics-cloud/how-to-monitor-k8s.html)
```
Comment (AndrewChubatiuk) [inline docs/guides/README.md:9]:
```suggestion
1. [How to monitor kubernetes cluster using VictoriaMetrics Cloud](../victoriametrics-cloud/how-to-monitor-k8s.md)
```

File: docs/guides/understand-your-setup-size.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ menu:
 aliases:
 - /guides/understand-your-setup-size.html
 ---
-The docs provide a simple and high-level overview of Ingestion Rate, Active Time Series, and Query per Second. These terms are a part of capacity planning ([Single-Node](https://docs.victoriametrics.com/single-server-victoriametrics/#capacity-planning), [Cluster](https://docs.victoriametrics.com/cluster-victoriametrics/#capacity-planning)) and [Managed VictoriaMetrics](https://docs.victoriametrics.com/managed-victoriametrics/) pricing.
+The docs provide a simple and high-level overview of Ingestion Rate, Active Time Series, and Query per Second. These terms are a part of capacity planning ([Single-Node](https://docs.victoriametrics.com/single-server-victoriametrics/#capacity-planning), [Cluster](https://docs.victoriametrics.com/cluster-victoriametrics/#capacity-planning)) and [VictoriaMetrics Cloud](https://docs.victoriametrics.com/victoriametrics-cloud/) pricing.
```
Comment (AndrewChubatiuk) [inline docs/guides/understand-your-setup-size.md:11]:
```suggestion
The docs provide a simple and high-level overview of Ingestion Rate, Active Time Series, and Query per Second. These terms are a part of capacity planning ([Single-Node](../Single-server-VictoriaMetrics.md#capacity-planning), [Cluster](../Cluster-VictoriaMetrics.md#capacity-planning)) and [VictoriaMetrics Cloud](../victoriametrics-cloud/README.md) pricing.
```

File: docs/victoriametrics-cloud/overview.md [hunk 4]
```diff
@@ -19,20 +22,20 @@ The most common use cases for VictoriaMetrics are:
...
-* Built-in [Alerting & Recording](https://docs.victoriametrics.com/managed-victoriametrics/alertmanager-setup-for-deployment/#configure-alerting-rules) rules execution;
-* Hosted [Alertmanager](https://docs.victoriametrics.com/managed-victoriametrics/alertmanager-setup-for-deployment/) for sending notifications;
-* Every Managed VictoriaMetrics deployment runs in an isolated environment, so deployments cannot interfere with each other;
-* Managed VictoriaMetrics deployment can be scaled up or scaled down in a few clicks;
+* It can be used as a Managed Prometheus - just configure Prometheus or vmagent to write data to VictoriaMetrics Cloud and then use the provided endpoint as a Prometheus datasource in Grafana;
+* Built-in [Alerting & Recording](https://docs.victoriametrics.com/victoriametrics-cloud/alertmanager-setup-for-deployment/#configure-alerting-rules) rules execution;
+* Hosted [Alertmanager](https://docs.victoriametrics.com/victoriametrics-cloud/alertmanager-setup-for-deployment/) for sending notifications;
```
Comment (AndrewChubatiuk) [inline docs/victoriametrics-cloud/overview.md:37]:
```suggestion
* Hosted [Alertmanager](./alertmanager-setup-for-deployment.md) for sending notifications;
```

File: docs/victoriametrics-cloud/overview.md [hunk 5]
```diff
@@ -19,20 +22,20 @@ The most common use cases for VictoriaMetrics are:
...
+VictoriaMetrics Cloud comes with the following features:
 
-* It can be used as a Managed Prometheus - just configure Prometheus or vmagent to write data to Managed VictoriaMetrics and then use the provided endpoint as a Prometheus datasource in Grafana;
-* Built-in [Alerting & Recording](https://docs.victoriametrics.com/managed-victoriametrics/alertmanager-setup-for-deployment/#configure-alerting-rules) rules execution;
-* Hosted [Alertmanager](https://docs.victoriametrics.com/managed-victoriametrics/alertmanager-setup-for-deployment/) for sending notifications;
-* Every Managed VictoriaMetrics deployment runs in an isolated environment, so deployments cannot interfere with each other;
-* Managed VictoriaMetrics deployment can be scaled up or scaled down in a few clicks;
+* It can be used as a Managed Prometheus - just configure Prometheus or vmagent to write data to VictoriaMetrics Cloud and then use the provided endpoint as a Prometheus datasource in Grafana;
+* Built-in [Alerting & Recording](https://docs.victoriametrics.com/victoriametrics-cloud/alertmanager-setup-for-deployment/#configure-alerting-rules) rules execution;
```
Comment (AndrewChubatiuk) [inline docs/victoriametrics-cloud/overview.md:36]:
```suggestion
* Built-in [Alerting & Recording](./alertmanager-setup-for-deployment.md#configure-alerting-rules) rules execution;
```

File: docs/victoriametrics-cloud/how-to-monitor-k8s.md [hunk 6]
```diff
@@ -24,7 +27,7 @@ This chart will install `VMOperator`, `VMAgent`, `NodeExporter`, `kube-state-met
 
 ## Prerequisites
 
-- Active Managed VictoriaMetrics instance. You can learn how to sign up for Managed VictoriaMetrics [here](https://docs.victoriametrics.com/managed-victoriametrics/quickstart.html#how-to-register). 
+- Active VictoriaMetrics Cloud instance. You can learn how to sign up for VictoriaMetrics Cloud [here](https://docs.victoriametrics.com/victoriametrics-cloud/quickstart.html#how-to-register).
```
Comment (AndrewChubatiuk) [inline docs/victoriametrics-cloud/how-to-monitor-k8s.md:30]:
```suggestion
- Active VictoriaMetrics Cloud instance. You can learn how to sign up for VictoriaMetrics Cloud [here](./quickstart.md#how-to-register). 
```

File: docs/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.md [hunk 7]
```diff
@@ -45,22 +49,22 @@ groups:
...
 ```
 
-### Managed VictoriaMetrics access token and deployment endpoint
+### VictoriaMetrics Cloud access token and deployment endpoint
 
-To use vmalert with Managed VictoriaMetrics, you must create a read/write token, or use an existing one. The token must have write access to ingest recording rules, ALERTS and ALERTS_FOR_STATE metrics, and read access for rules evaluation.
+To use vmalert with VictoriaMetrics Cloud, you must create a read/write token, or use an existing one. The token must have write access to ingest recording rules, ALERTS and ALERTS_FOR_STATE metrics, and read access for rules evaluation.
 
-For instructions on how to create tokens, please refer to this section of the [documentation](https://docs.victoriametrics.com/managed-victoriametrics/quickstart.html#deployment-access).
+For instructions on how to create tokens, please refer to this section of the [documentation](https://docs.victoriametrics.com/victoriametrics-cloud/quickstart.html#deployment-access).
```
Comment (AndrewChubatiuk) [inline docs/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.md:56]:
```suggestion
For instructions on how to create tokens, please refer to this section of the [documentation](./quickstart.md#deployment-access).
```

File: docs/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.md [hunk 8]
```diff
@@ -1,23 +1,27 @@
...
+This guide explains the different ways in which you can use vmalert in conjunction with VictoriaMetrics Cloud
+
+![Metrics setup](alerting-vmalert-victoria-metrics-cloud_setup.webp)
 
 ## Preconditions 
 
 * [vmalert](https://docs.victoriametrics.com/vmalert/) is installed. You can obtain it by building it from [source](https://docs.victoriametrics.com/vmalert/#quickstart), downloading it from the [GitHub releases page](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/latest), or using the [docker image](https://hub.docker.com/r/victoriametrics/vmalert) for the container ecosystem (such as docker, k8s, etc.).
 * [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/) is installed.
-* You have a [single or cluster](https://docs.victoriametrics.com/managed-victoriametrics/quickstart.html#creating-deployment) deployment in [Managed VictoriaMetrics](https://docs.victoriametrics.com/managed-victoriametrics/overview.html).
+* You have a [single or cluster](https://docs.victoriametrics.com/victoriametrics-cloud/quickstart.html#creating-deployment) deployment in [VictoriaMetrics Cloud](https://docs.victoriametrics.com/victoriametrics-cloud/overview.html).
```
Comment (AndrewChubatiuk) [inline docs/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.md:24]:
```suggestion
* You have a [single or cluster](./quickstart.md#creating-deployment) deployment in [VictoriaMetrics Cloud](./overview.md).
```

File: docs/victoriametrics-cloud/README.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
+* [Overview of VictoriaMetrics Cloud](/victoriametrics-cloud/overview/)
```
Comment (AndrewChubatiuk) [inline docs/victoriametrics-cloud/README.md:1]:
```suggestion
* [Overview of VictoriaMetrics Cloud](./overview.md)
```

File: docs/victoriametrics-cloud/README.md [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+* [Quick Start](/victoriametrics-cloud/quickstart/)
```
Comment (AndrewChubatiuk) [inline docs/victoriametrics-cloud/README.md:2]:
```suggestion
* [Quick Start](./quickstart.md)
```

File: docs/victoriametrics-cloud/README.md [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+
+## Guides
+* [Understand Your Setup Size](/guides/understand-your-setup-size/)
```
Comment (AndrewChubatiuk) [inline docs/victoriametrics-cloud/README.md:5]:
```suggestion
* [Understand Your Setup Size](../guides/understand-your-setup-size.md)
```

File: docs/victoriametrics-cloud/README.md [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+* [Alerting & recording rules with Alertmanager configuration for VictoriaMetrics Cloud deployment](/victoriametrics-cloud/alertmanager-setup-for-deployment/)
```
Comment (AndrewChubatiuk) [inline docs/victoriametrics-cloud/README.md:6]:
```suggestion
* [Alerting & recording rules with Alertmanager configuration for VictoriaMetrics Cloud deployment](./alertmanager-setup-for-deployment.md)
```

File: docs/victoriametrics-cloud/README.md [hunk 13]
```diff
@@ -0,0 +1,14 @@
...
+* [Kubernetes Monitoring with VictoriaMetrics Cloud](/victoriametrics-cloud/how-to-monitor-k8s/)
```
Comment (AndrewChubatiuk) [inline docs/victoriametrics-cloud/README.md:7]:
```suggestion
* [Kubernetes Monitoring with VictoriaMetrics Cloud](./how-to-monitor-k8s.md)
```

File: docs/victoriametrics-cloud/README.md [hunk 14]
```diff
@@ -0,0 +1,14 @@
...
+* [Setup Notifications](/victoriametrics-cloud/setup-notifications/)
```
Comment (AndrewChubatiuk) [inline docs/victoriametrics-cloud/README.md:8]:
```suggestion
* [Setup Notifications](./setup-notifications.md)
```

File: docs/victoriametrics-cloud/README.md [hunk 15]
```diff
@@ -0,0 +1,14 @@
...
+* [User Management](/victoriametrics-cloud/user-managment/)
```
Comment (AndrewChubatiuk) [inline docs/victoriametrics-cloud/README.md:9]:
```suggestion
* [User Management](./user-managment.md)
```

File: docs/guides/understand-your-setup-size.md [hunk 16]
```diff
@@ -101,12 +101,12 @@ You have a Kubernetes environment that produces 5k time series per second with 1
 VictoriaMetrics requires additional disk space for the index. The lower Churn Rate means lower disk space usage for the index because of better compression.
 Usually, the index takes about 20% of the disk space for storing data. High cardinality setups may use >50% of datapoints storage size for index.
 
-You can significantly reduce the amount of disk usage by specifying [Downsampling](https://docs.victoriametrics.com/#downsampling) and [Retention Filters](https://docs.victoriametrics.com/#retention-filters) that are lower than the Retention Period. Both two settings are available in Managed VictoriaMetrics and Enterprise.
+You can significantly reduce the amount of disk usage by specifying [Downsampling](https://docs.victoriametrics.com/#downsampling) and [Retention Filters](https://docs.victoriametrics.com/#retention-filters) that are lower than the Retention Period. Both two settings are available in VictoriaMetrics Cloud and Enterprise.
```
Comment (AndrewChubatiuk) [inline docs/guides/understand-your-setup-size.md:104]:
```suggestion
You can significantly reduce the amount of disk usage by specifying [Downsampling](../#downsampling) and [Retention Filters](../#retention-filters) that are lower than the Retention Period. Both two settings are available in VictoriaMetrics Cloud and Enterprise.
```

**Review by AndrewChubatiuk — CHANGES_REQUESTED** [review]

File: docs/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.md [hunk 17]
```diff
@@ -1,23 +1,27 @@
...
+title: Alerting with vmalert and VictoriaMetrics Cloud
 menu:
   docs:
-    parent: "managed"
+    parent: "cloud"
     weight: 5
 aliases:
-- /managed-victoriametrics/alerting-vmalert-managed-victoria-metrics.html
+  - /victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.html
+  - /victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud/
```
Comment (tenmozes) [inline docs/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.md:11]: /victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud/index.html

Comment (tenmozes) [reply docs/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.md:11]: and please change this in all depended docs

File: docs/enterprise.md (hunk 1 above)
Comment (dmitryk-dk) [reply docs/enterprise.md]: done

File: docs/guides/README.md (hunk 2 above)
Comment (dmitryk-dk) [reply docs/guides/README.md]: done

File: docs/guides/understand-your-setup-size.md (hunk 3 above)
Comment (dmitryk-dk) [reply docs/guides/understand-your-setup-size.md]: done

File: docs/victoriametrics-cloud/overview.md (hunk 5 above)
Comment (dmitryk-dk) [reply docs/victoriametrics-cloud/overview.md]: done

File: docs/victoriametrics-cloud/overview.md (hunk 4 above)
Comment (dmitryk-dk) [reply docs/victoriametrics-cloud/overview.md]: done

File: docs/victoriametrics-cloud/how-to-monitor-k8s.md (hunk 6 above)
Comment (dmitryk-dk) [reply docs/victoriametrics-cloud/how-to-monitor-k8s.md]: done

File: docs/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.md (hunk 7 above)
Comment (dmitryk-dk) [reply docs/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.md]: done

File: docs/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.md (hunk 8 above)
Comment (dmitryk-dk) [reply docs/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.md]: done

File: docs/victoriametrics-cloud/README.md (hunk 10 above)
Comment (dmitryk-dk) [reply docs/victoriametrics-cloud/README.md]: done

File: docs/victoriametrics-cloud/README.md (hunk 11 above)
Comment (dmitryk-dk) [reply docs/victoriametrics-cloud/README.md]: done

File: docs/victoriametrics-cloud/README.md (hunk 12 above)
Comment (dmitryk-dk) [reply docs/victoriametrics-cloud/README.md]: done

File: docs/victoriametrics-cloud/README.md (hunk 13 above)
Comment (dmitryk-dk) [reply docs/victoriametrics-cloud/README.md]: done

File: docs/victoriametrics-cloud/README.md (hunk 14 above)
Comment (dmitryk-dk) [reply docs/victoriametrics-cloud/README.md]: done

File: docs/victoriametrics-cloud/README.md (hunk 15 above)
Comment (dmitryk-dk) [reply docs/victoriametrics-cloud/README.md]: done

File: docs/guides/understand-your-setup-size.md (hunk 16 above)
Comment (dmitryk-dk) [reply docs/guides/understand-your-setup-size.md]: done

File: docs/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.md (hunk 17 above)
Comment (dmitryk-dk) [reply docs/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.md]: updated for all victoriametrics-cloud folder

**Review by AndrewChubatiuk — APPROVED** [review]

File: docs/FAQ.md [hunk 18]
```diff
@@ -215,11 +215,11 @@ We provide commercial support for both versions. [Contact us](mailto:info@victor
...
 The following commercial versions of VictoriaMetrics are available:
 
-* [Managed VictoriaMetrics at AWS](https://aws.amazon.com/marketplace/pp/prodview-4tbfq5icmbmyc) (aka managed Prometheus).
+* [VictoriaMetrics Cloud at AWS](https://aws.amazon.com/marketplace/pp/prodview-4tbfq5icmbmyc) (aka managed Prometheus).
 
 The following commercial versions of VictoriaMetrics are planned:
 
-* Managed VictoriaMetrics at Google Cloud.
+* VictoriaMetrics Cloud at Google Cloud.
 * Cloud monitoring solution based on VictoriaMetrics.
```
Comment (yatskevich) [inline docs/FAQ.md:223]: Let's remove these lines

File: docs/Quick-Start.md [hunk 19]
```diff
@@ -20,7 +20,7 @@ VictoriaMetrics is distributed in two forms:
 
 Single-server-VictoriaMetrics VictoriaMetrics is available as:
 
-* [Managed VictoriaMetrics at AWS](https://aws.amazon.com/marketplace/pp/prodview-4tbfq5icmbmyc)
+* [VictoriaMetrics Cloud at AWS](https://aws.amazon.com/marketplace/pp/prodview-4tbfq5icmbmyc)
```
Comment (yatskevich) [inline docs/Quick-Start.md:23]: `[VictoriaMetrics Cloud](https://cloud.victoriametrics.com/signUp?utm_source=website&utm_campaign=docs_vm_quickstart) – the most cost-efficient hosted monitoring platfor, operated by VictoriaMetrics core team.`

File: docs/victoriametrics-cloud/README.md [hunk 20]
```diff
@@ -0,0 +1,14 @@
...
+
+## Guides
+* [Understand Your Setup Size](../guides/understand-your-setup-size.md)
+* [Alerting & recording rules with Alertmanager configuration for VictoriaMetrics Cloud deployment](./alertmanager-setup-for-deployment.md)
+* [Kubernetes Monitoring with VictoriaMetrics Cloud](./how-to-monitor-k8s.md)
+* [Setup Notifications](./setup-notifications.md)
+* [User Management](./user-managment.md)
+
+Learn more about VictoriaMetrics Cloud:
+* [VictoriaMetrics Cloud announcement](https://victoriametrics.com/blog/victoriametrics-cloud-announcement)
```
Comment (yatskevich) [inline docs/victoriametrics-cloud/README.md:12]: The link returns 404. It should be `https://victoriametrics.com/blog/managed-victoriametrics-announcement`

File: docs/victoriametrics-cloud/how-to-monitor-k8s.md [hunk 21]
```diff
@@ -24,7 +25,7 @@ This chart will install `VMOperator`, `VMAgent`, `NodeExporter`, `kube-state-met
 
 ## Prerequisites
 
-- Active Managed VictoriaMetrics instance. You can learn how to sign up for Managed VictoriaMetrics [here](https://docs.victoriametrics.com/managed-victoriametrics/quickstart.html#how-to-register). 
+- Active VictoriaMetrics Cloud instance. You can learn how to sign up for VictoriaMetrics Cloud [here](./quickstart.md#how-to-register).
```
Comment (yatskevich) [inline docs/victoriametrics-cloud/how-to-monitor-k8s.md:28]: `Active VictoriaMetrics Cloud instance` -> `Active VictoriaMetrics Cloud deployment`

**Review by yatskevich — CHANGES_REQUESTED** [review]

File: docs/victoriametrics-cloud/README.md (hunk 20 above)
Comment (dmitryk-dk) [reply docs/victoriametrics-cloud/README.md:12]: As far as i can see it should be https://victoriametrics.com/blog/introduction-to-managed-monitoring/

File: docs/FAQ.md [hunk 22]
```diff
@@ -215,11 +215,11 @@ We provide commercial support for both versions. [Contact us](mailto:info@victor
 
 The following commercial versions of VictoriaMetrics are available:
 
-* [Managed VictoriaMetrics at AWS](https://aws.amazon.com/marketplace/pp/prodview-4tbfq5icmbmyc) (aka managed Prometheus).
+* [VictoriaMetrics Cloud at AWS](https://aws.amazon.com/marketplace/pp/prodview-4tbfq5icmbmyc) (aka managed Prometheus).
```
[the remark this answers is not in the document]
Comment (dmitryk-dk) [reply docs/FAQ.md]: fixed

File: docs/FAQ.md (hunk 18 above)
Comment (dmitryk-dk) [reply docs/FAQ.md]: fixed

File: docs/Quick-Start.md (hunk 19 above)
Comment (dmitryk-dk) [reply docs/Quick-Start.md]: fixed

**Review by Amper — APPROVED** [review]
LGTM

**Review by yatskevich — APPROVED** [review]
LGTM

**Pull request merged**